static/js/lib/api_test.js

```diff
@@ -580,7 +573,7 @@ describe('api', function() {
           });
 
           return assert.isRejected(fetchJSONWithCSRF('/url', {}, true)).then(() => {
-            assert.equal(savedWindowLocation, '/login/edxorg/');
+            assert.isTrue(window.location.toString().includes('/login/edxorg/'));
```

Comment:
[`assert.include`](http://chaijs.com/api/assert/#method_include) would be clearer, and gets you a nicer error message:
```js
assert.include(window.location.toString(), '/login/edxorg/');
```